core/drivers/crypto/se050/adaptors/include/se050_default_keys.h

```diff
@@ -0,0 +1,123 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2020 - All Rights Reserved
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#ifndef SE05X_DEFAULT_KEYS_H_
+#define SE05X_DEFAULT_KEYS_H_
+
+#include <se050_utils.h>
+#include <stdint.h>
+
+/* OF ID */
+#define SE050A1_ID 0xA204
+#define SE050A2_ID 0xA205
+#define SE050B1_ID 0xA202
+#define SE050B2_ID 0xA203
+#define SE050C1_ID 0xA200
+#define SE050C2_ID 0xA201
+#define SE050DV_ID 0xA1F4
+
+#define SE050A1 0
+#define SE050A2 1
+#define SE050B1 2
+#define SE050B2 3
+#define SE050C1 4
+#define SE050C2 5
+#define SE050DV 6
+
+static inline sss_status_t se050_get_id_from_ofid(uint32_t ofid, uint32_t *id)
+{
+	sss_status_t status = kStatus_SSS_Success;
+
+	switch (ofid) {
+	case SE050A1_ID:
+		*id = 0;
+		break;
+	case SE050A2_ID:
+		*id = 1;
+		break;
+	case SE050B1_ID:
+		*id = 2;
+		break;
+	case SE050B2_ID:
+		*id = 3;
+		break;
+	case SE050C1_ID:
+		*id = 4;
+		break;
+	case SE050C2_ID:
+		*id = 5;
+		break;
+	case SE050DV_ID:
+		*id = 6;
+		break;
+	default:
+		EMSG("invalid ofid");
+		status = kStatus_SSS_Fail;
```

Comment:
simply `return kStatus_SSS_Fail` here and `return kStatus_SSS_Success` at line 61.